runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -46,6 +46,8 @@
 import static net.runelite.client.plugins.cluescrolls.clues.emote.Emote.BULL_ROARER;
 import net.runelite.client.plugins.cluescrolls.clues.emote.ItemRequirement;
 import net.runelite.client.plugins.cluescrolls.clues.emote.RangeItemRequirement;
+import static net.runelite.client.plugins.cluescrolls.clues.emote.STASHUnit.*;
+import static net.runelite.client.plugins.cluescrolls.clues.emote.STASHUnit.SHANTAY_PASS;
```

Comment:
Don't need this if also using a star import.

Reply:
Its needed because there is another `SHANTAY_PASS` (from `ItemID`) in scope